modules/local/propr/grea/main.nf

```diff
@@ -3,9 +3,9 @@ process PROPR_GREA {
     label 'process_high'
 
     // conda "${moduleDir}/environment.yml"
```

Comment:
Why is conda not an option for this module?

Reply:
so the module is temporal yet, 
I plan to finish the nf-corization of the module during the hackathon, and by then the conda environment and all the nf-tests should be ready